test/run_test.js

```diff
@@ -1,22 +1,24 @@
 import { runTasks } from '../lib/run'
-// import { expect } from 'chai'
-// import { stub } from 'sinon'
-
-xdescribe('lib/run', () => {
+import childProcess from 'child_process'
+import { expect } from 'chai'
+import { spy } from 'sinon'
 
+describe('lib/run', () => {
   describe('runTasks', () => {
-    const tasks = [
-      'this:thing',
-      'that:thing'
-    ]
+    const tasks = ['dummy:1', 'dummy:2']
 
     before(() => {
+      spy(childProcess, 'exec')
       return runTasks(tasks)
     })
 
-    it('executes an `npm run` for each task', () => {
+    after(() => {
+      childProcess.exec.restore()
     })
 
+    it('executes an `npm run` for each task', () => {
+      expect(childProcess.exec).to.have.been.calledWith(`npm run dummy:1`)
```

Comment:
could iterate over tasks and assert, like `tasks.forEach(task => expect()...with(`npm run${task}`)`